com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md

```diff
@@ -0,0 +1,44 @@
+# Reflection probe usage
+
+## What is a probe
+
+Reflection probes and planar reflection probes provide indirect specular lighting.
```

Comment:
[Reflection Probes](Reflection-Probe.md) and [Planar Reflection Probes](Planar-Reflection-Probe.md) provide indirect specular lighting.